dist/index.js

```diff
@@ -1815,7 +1815,9 @@ ${diff}
 `);
         const output = scrubSecrets(`
 ## ArgoCD Diff for commit [\`${shortCommitSha}\`](${commitLink})
-_Updated at ${new Date().toLocaleString('en-US', { timeZone: 'America/Los_Angeles' })} PT_
+${ARGOCD_SERVER_URL}
```

Comment:
Can you format this a little bit

`cluster: https://${ARGOCD_SERVER_URL}`  (or target or if something else make more sense that :p)